application/overlay/_twig/Jenkinsfile/test.twig

```diff
@@ -0,0 +1,12 @@
+        stage('Test')  {
+            parallel {
+{% if not @('jenkins.tests.isolated') %}
+{% if @('services.lighthouse.enabled') %}
```

Comment:
Does this need bool()?
```suggestion
{% if bool(@('services.lighthouse.enabled')) %}
```

Reply:
no, bool is a php harness thing as docker harness doesn't use yes's ever

Reply:
we could say though that bool is a nice type check in any case to avoid WTFs